Sources/TransferMethod/SelectTransferMethodTypeCell.swift

```diff
@@ -28,14 +28,9 @@ struct SelectTransferMethodTypeConfiguration {
 final class SelectTransferMethodTypeCell: UITableViewCell {
```

Comment:
Suggestion:  The `ListTransferMethodTableViewCell ` and `SelectTransferMethodTypeCell ` except for the `configure` in the `detailTextLabel?.attributedText = ...` part.  Maybe we could create a common `UITableViewCell`

Reply:
We can create another ticket for it.